gluon/restricted.py

```diff
@@ -199,7 +199,7 @@ def __str__(self):
 
 
 def compile2(code, layer):
-    return compile(code.rstrip(), layer, 'exec')
+    return compile(code, layer, 'exec')
```

Comment:
Since web2py supports py27+ according to https://docs.python.org/2/library/functions.html?highlight=compile#compile  rstrip is no longer necessary on both mac and win